rpcs3/Emu/Cell/SPUASMJITRecompiler.cpp

```diff
@@ -1593,6 +1593,35 @@ void spu_recompiler::RCHCNT(spu_opcode_t op)
 		spu_log.warning("[0x%x] RCHCNT: RdEventStat", m_pos);
 		[[fallthrough]]; // fallback
```

Comment:
Do the new return 1 cases cause a problem here? (since the fallthrough now hits return 1 rather than the previous default:)

Reply:
get_ch_count() handles all counts properly so no.